.github/workflows/swift-toolchain.yml

```diff
@@ -73,7 +73,14 @@ jobs:
           sudo DEBIAN_FRONTEND=noninteractive apt-get -qq update -yq
           sudo DEBIAN_FRONTEND=noninteractive apt-get -qq -o Dpkg::Use-Pty=0 install -yq repo libxml2-utils
 
-          repo init --quiet --groups default --depth 1 -u https://github.com/compnerd/swift-build -b main
+          # Which branch is this workflow based on
+          branch_version=main
+          if [[ "$branch_version" == "main" ]] ; then
+            branch_name="main"
+          else
+            branch_name="release/${branch_version}"
```

Comment:
I think that we should be able to use the github variable for the branch names.  We follow the same branch names apple does.  This means that we should just drop the `release/` prefix on the branch name.